contracts/src/core/EigenDACertVerifier.sol

```diff
@@ -50,164 +28,19 @@ contract EigenDACertVerifier is IEigenDACertVerifier {
-    ) external view {
-        CertV2Lib.verifyDACertV2FromSignedBatch(
-            eigenDAThresholdRegistry,
-            eigenDASignatureVerifier,
-            eigenDARelayRegistry,
-            operatorStateRetriever,
-            registryCoordinator,
-            signedBatch,
-            blobInclusionInfo,
-            securityThresholdsV2,
-            quorumNumbersRequiredV2
-        );
-    }
-
-    /**
-     * @notice Thin try/catch wrapper around verifyDACertV2 that returns false instead of panicing
-     * @dev The Steel library (https://github.com/risc0/risc0-ethereum/tree/main/crates/steel)
-     *      currently has a limitation that it can only create zk proofs for functions that return a value
-     * @param batchHeader The batch header of the blob
-     * @param blobInclusionInfo The inclusion proof for the blob cert
-     * @param nonSignerStakesAndSignature The nonSignerStakesAndSignature to verify the blob cert against
-     * @param signedQuorumNumbers The signed quorum numbers corresponding to the nonSignerStakesAndSignature
-     */
-    function verifyDACertV2ForZKProof(
-        BatchHeaderV2 calldata batchHeader,
-        BlobInclusionInfo calldata blobInclusionInfo,
-        NonSignerStakesAndSignature calldata nonSignerStakesAndSignature,
-        bytes memory signedQuorumNumbers
-    ) external view returns (bool) {
-        (CertV2Lib.StatusCode status,) = CertV2Lib.checkDACertV2(
-            eigenDAThresholdRegistry,
-            eigenDASignatureVerifier,
-            eigenDARelayRegistry,
-            batchHeader,
-            blobInclusionInfo,
-            nonSignerStakesAndSignature,
-            securityThresholdsV2,
-            quorumNumbersRequiredV2,
-            signedQuorumNumbers
-        );
-        if (status == CertV2Lib.StatusCode.SUCCESS) {
-            return true;
-        } else {
-            return false;
-        }
-    }
-
-    ///////////////////////// HELPER FUNCTIONS ///////////////////////////////
-
-    /**
-     * @notice Returns the nonSignerStakesAndSignature for a given blob cert and signed batch
-     * @param signedBatch The signed batch to get the nonSignerStakesAndSignature for
-     * @return nonSignerStakesAndSignature The nonSignerStakesAndSignature for the given signed batch attestation
-     */
-    function getNonSignerStakesAndSignature(SignedBatch calldata signedBatch)
-        external
-        view
-        returns (NonSignerStakesAndSignature memory)
-    {
-        (NonSignerStakesAndSignature memory nonSignerStakesAndSignature,) =
-            CertV2Lib.getNonSignerStakesAndSignature(operatorStateRetriever, registryCoordinator, signedBatch);
-        return nonSignerStakesAndSignature;
-    }
-
-    /**
-     * @notice Verifies the security parameters for a blob cert
-     * @param blobParams The blob params to verify
-     * @param securityThresholds The security thresholds to verify against
-     */
     function verifyDACertSecurityParams(
```

Comment:
Did you intentionally delete the function docs from all these functions?

Reply:
No, but this contract is slated for deletion in a future PR anyway, as well as most functions in this contract in general. With that context, do you think it would be necessary to fix this?

Reply:
I'd argue its a quick fix and worth not introducing a regression here if possible

Reply:
https://github.com/Layr-Labs/eigenda/pull/1492/commits/82caf341a8ebb281cf9149fd6008f998856c8a1a